crates/re_types/definitions/rerun/components/pinhole_projection.fbs

```diff
@@ -18,7 +18,7 @@ namespace rerun.components;
 ///    0.0     0.0    1.0
 /// ```
 struct PinholeProjection (
-  "attr.rust.derive": "Copy, PartialEq, PartialOrd"
+  "attr.rust.derive": "Default, Copy, PartialEq, PartialOrd"
```

Comment:
I don't know what the default `Mat3x3` is (Zero? Identity?) but neither one is a very good default for `PinholeProjection`

Reply:
It's identity. Not sure if I can come up with a better one 🤔